src/shared/actions/alerts.js

```diff
@@ -59,6 +60,34 @@ export const generateAlert = (category, title, message, closeInterval, err) => (
     }
 };
 
+
+/**
+ * Generates relevant node error alert
+ *
+ * @method generateNodeError
+ * @param {func} generateDefaultAlert
+ * @param {object} err
+ *
+ * @returns {function} dispatch
+ */
+export const generateNodeError = (generateDefaultAlert, err) => (dispatch) => {
+    if (get(err, 'message') === Errors.LEDGER_CANCELLED) {
+        dispatch(generateLedgerCancelledAlert(err));
+    } else if (get(err, 'message') === Errors.LEDGER_INVALID_INDEX) {
```

Comment:
I think it's best to make another function for Ledger-related errors instead of grouping them with the node errors